src/hacbs/components/Environment/EnvironmentListView.tsx

```diff
@@ -24,6 +25,7 @@ type Props = {
 const EnvironmentListView: React.FC<Props> = ({
   validTypes = Object.keys(EnvironmentType).map((t) => EnvironmentType[t]),
 }) => {
+  const [allEnvironments] = useSortedEnvironments();
```

Comment:
Not ideal that both the parent component uses `useSortedEnvironments` and the `EnvironmentListViewBase` recomputes this again.